test/browser/amd/index.html

```diff
@@ -1,43 +0,0 @@
-<!DOCTYPE html>
```

Comment:
You're removing my primary means of testing these changes in SpiderMonkey-based engines. Automatic r- for that alone.